src/app/components/ScrollLock.tsx

```diff
@@ -36,8 +37,16 @@ const fadeIn = keyframes`
   }
 `;
 
+interface OverlayProps {
+  className?: string;
+  [key: string]: any;
+}
+
 // tslint:disable-next-line:variable-name
-export const Overlay = styled.div`
+export const Overlay = styled(({ className, ...otherProps }: OverlayProps) => {
+  useDisableContentTabbing();
+  return <div className={className} {...otherProps} />;
```

Comment:
you don't have to split out the classname it could be applied the same way as `otherProps`